README.md

```diff
@@ -94,6 +95,20 @@ When `storage.blocks.type` is `S3` the instance will use the S3 AWS service for
 
 Additionally, the following environment variables must be present: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION`.
 
+### Keys Config
+
+Kepler hosts require key pairs to provide replication. The `keys` config fields specify how a Kepler instance generates and stores these key pairs.
+
+#### Static Secret Derivation
+
+When `keys.type` is `Static` the instance will use an array of bytes as a static secret from which it will derive key pairs on a per-Orbit basis. The following config options will be available:
+
+| Option      | env var            | description                                                                  |
+|:------------|:-------------------|:-----------------------------------------------------------------------------|
+| keys.secret | KEPLER_KEYS_SECRET | Unpadded base64Url-encoded byte string from which key pairs will be derived. |
+
+It is STRONGLY RECOMMENDED that the secret be given via environment variables and NOT in the `kepler.toml` config file.
```

Comment:
Probably should strongly recommend to backup the secret as well. Is there a rotation mechanism in case it gets compromised?

Reply:
Good point, and yeah there's no rotation mechanism except for an orbit controller to revoke the old orbit host permission and issue a new one. for rotation of the secret itself, I'm not sure how one could work given it's just derived via hash.

Reply:
secret is not longer randomly generated if missing from the config, to prevent users from forgetting to generate and back it up. An error will be thrown if it's not provided